config/test_cases/test_pull_request_openshift_ansible_extended_conformance_install_with_status_check.yml

```diff
@@ -7,12 +7,10 @@ overrides:
       type: "pull_request"
     - name: "origin"
 extensions:
-  sync_repos:
-    - name: "aos-cd-jobs"
   actions:
     - type: "host_script"
       title: "check other job statuses"
-      repository: "aos-cd-jobs"
       script: |-
-        git checkout sjb
+        wget https://raw.githubusercontent.com/openshift/aos-cd-jobs/sjb/check-pull-request-status.py
```

Comment:
I'd rather use `scp` to grab them from the VM -- we know the SHA from which that repo was cloned there, etc -- this is a little less inspectable.